CHANGELOG.md

```diff
@@ -1,19 +1,23 @@
+## v2.1.0
+* Did a major refactor. Now async and sync WebDriver are fully supported
+  for JsonWire and W3C spec. They also share the spec related code now.
```

Comment:
What does "They also share the spec related code now" mean? What is the spec related code?